client/src/store/new-request-module.ts

```diff
@@ -1791,27 +1797,27 @@ export class NewRequestModule extends VuexModule {
   }
   @Action
   async getNameAnalysis () {
-    this.mutateDisplayedComponent('AnalyzePending')
-    this.resetRequestExaminationOrProvideConsent()
-
-    let params: NewRequestNameSearchI = {
-      name: this.name,
-      location: this.location,
-      // @ts-ignore TODO: This is not typed correctly!
-      entity_type_cd: this.entity_type_cd,
-      request_action_cd: this.request_action_cd
-    }
-
     try {
-      let { CancelToken } = axios
-      source = CancelToken.source()
+      this.mutateDisplayedComponent('AnalyzePending')
+      this.resetRequestExaminationOrProvideConsent()
+
+      const params: NewRequestNameSearchI = {
+        name: this.name,
+        location: this.location,
+        // @ts-ignore TODO: This is not typed correctly!
+        entity_type_cd: this.entity_type_cd,
+        request_action_cd: this.request_action_cd
+      }
+      const { CancelToken } = axios
+      const source = CancelToken.source()
```

Comment:
I actually didn't change much here... please hide whitespace changes to see a cleaner diff.